lib/jazzy/sourcekitten.rb

```diff
@@ -448,6 +458,7 @@ def self.autolink(docs, root_decls)
         doc.abstract = autolink_text(doc.abstract, doc, root_decls)
         doc.return = autolink_text(doc.return, doc, root_decls) if doc.return
         doc.children = autolink(doc.children, root_decls)
+        doc.declaration &&= autolink_text(doc.declaration, doc, root_decls, true)
```

Comment:
Ok perhaps this could be rewritten to be consisent with the above `doc.return = autolink(..) if doc.return` style?